app/jobs/setup_metadata_job.rb

```diff
@@ -53,7 +56,11 @@ def setup_child_object_jobs(parent_object, current_batch_process)
       if child.pyramidal_tiff.height_and_width? && S3Service.s3_exists?(child.remote_ptiff_path)
         child.processing_event("PTIFF exists on S3, not converting: #{child.oid}", 'ptiff-ready-skipped')
       else
-        GeneratePtiffJob.perform_later(child, current_batch_process)
+        path = Pathname.new(child.access_master_path)
+        file_size = File.exist?(path) ? File.size(path) : 0
+        # 1073741824 is 1GB in bytes
+        GeneratePtiffJob.set(queue: :large_ptiff).perform_later(child, current_batch_process) if file_size > 1_073_741_824
```

Comment:
I'd suggest defining a constant for the size value and referencing it here and in `batch_process.rb`

Reply:
Went ahead and updated this to use a constant.